bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
-from rdflib import Literal
-from .namespaces import SKOS, OWL, BRICK
+from brickschema.graph import Graph
+from brickschema.inference import BrickInferenceSession
+from rdflib import Literal, URIRef
+from .namespaces import SKOS, OWL, RDFS, BRICK, QUDTQK, QUDTDV, QUDT, UNIT
 
 
+g = Graph()
+g.load_file("support/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl")
+g.load_file("support/VOCAB_QUDT-UNITS-ALL-v2.1.ttl")
+g.g.bind("qudt", QUDT)
+g.g.bind("qudtqk", QUDTQK)
+sess = BrickInferenceSession()
+g = sess.expand(g)
+
+
+def get_units(brick_quantity):
+    """
+    Fetches the QUDT unit and symbol (as a Literal) from the QUDT ontology so
+    in order to avoid having to pull the full QUDT ontology into Brick
+    """
+    res = g.query(
+        f"""SELECT ?unit ?symbol WHERE {{
+                    <{brick_quantity}> qudt:applicableUnit ?unit .
+                    ?unit qudt:symbol ?symbol .
+                    FILTER(isLiteral(?symbol))
+                    }}"""
+    )
+    for r in res:
+        yield r
+
+
+"""
+Each is a qudt:QuantityKind
+"""
 quantity_definitions = {
     "Air_Quality": {
-        "subclasses": {
-            "CO2_Level": {},
-            "PM10_Level": {},
-            "PM25_Level": {},
-            "TVOC_Level": {},
+        "subconcepts": {
+            "CO2_Level": {
```

Comment:
I suggest changing `Level` to `Concentration` for all the current Level quantities. Level is vague, and Concentration is more explicit as in each of the documentations.